runtime/compiler/aarch64/codegen/J9TreeEvaluator.hpp

```diff
@@ -53,6 +53,8 @@ class OMR_EXTENSIBLE TreeEvaluator: public J9::TreeEvaluator
 
    static TR::Register *DIVCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg);
 
+   static TR::Register *arrayCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
The name of the evaluator is inconsistent with the .cpp file.  (`arrayCHK` or `ArrayCHK`).